marklogic-data-hub-central/ui/e2e/cypress/e2e/modeling/verifyConceptClassesInModeling.cy.tsx

```diff
@@ -365,4 +364,130 @@ describe("Concept classes in Modeling screen", () => {
     entityTypeTable.getConceptClass("TestConcept").should("not.exist");
   });
 
-});
+  it("Can delete concept class and relationship and publish without problems", () => {
+    cy.waitUntil(() => toolbar.getModelToolbarIcon()).click();
+    cy.wait(5000);
+    graphView.getAddButton().click();
+    graphView.getAddConceptClassOption().should("be.visible").click({force: true});
+    conceptClassModal.newConceptClassName("testDeleteStyle");
+    conceptClassModal.newConceptClassDescription("Created just to validate deleting.");
+    cy.waitUntil(() => conceptClassModal.getAddButton().click());
+    conceptClassModal.getAddButton().should("not.exist");
+
+    cy.wait(5000);
+    cy.log("**Create Concept Class**");
+    graphViewSidePanel.getSelectedConceptClassHeading("testDeleteStyle").should("exist");
+    graphViewSidePanel.getSelectedConceptClassHeadingInfo("testDeleteStyle").should("exist");
+    graphViewSidePanel.getConceptClassDescription().should("be.visible");
+    graphViewSidePanel.getConceptClassDescription().should("have.value", "Created just to validate deleting.");
+
+    modelPage.toggleColorSelector("testDeleteStyle");
+    modelPage.selectColorFromPicker("#D5D3DD").click();
+    if (Cypress.isBrowser("!firefox")) {
+      graphViewSidePanel.getConceptClassColor("testDeleteStyle").should("have.css", "background", "rgb(213, 211, 221) none repeat scroll 0% 0% / auto padding-box border-box");
+    }
+    if (Cypress.isBrowser("firefox")) {
+      graphViewSidePanel.getConceptClassColor("testDeleteStyle").should("have.css", "background-color", "rgb(213, 211, 221)");
+    }
+
+    modelPage.openIconSelector("testDeleteStyle");
+    modelPage.selectIcon("testDeleteStyle", "FaAccessibleIcon");
+    modelPage.getIconSelected("testDeleteStyle", "FaAccessibleIcon").should("exist");
+    graphViewSidePanel.closeSidePanel();
+    graphViewSidePanel.getSelectedConceptClassHeading("testDeleteStyle").should("not.exist");
+
+    cy.log("**Create a relationship with the created concept class**");
+    cy.waitUntil(() => toolbar.getModelToolbarIcon()).click();
```

Comment:
We are already in the model tile. In this scenario there is no need to click the tile icon